app/pb_kits/playbook/pb_charts/business_intelligence/pbLineGraph.js

```diff
@@ -0,0 +1,15 @@
+import pbHighcharts from "./pbHighcharts.js"
+
+const markerStyles = function(highchart){
+  highchart.plotOptions.line.marker.enabled = true;
+  highchart.plotOptions.series.marker.enabled = true;
+  highchart.plotOptions.series.marker.fillColor = "white";
+  highchart.plotOptions.series.marker.lineWidth = 2;
+}
+
+const pbLineGraph = function(highchart) {
+  markerStyles(highchart);
+  pbHighcharts(highchart);
```

Comment:
Should pbHighcharts() be called before markerStyles() ? 
Since markerStyles is a specific override for pbLineGraph, in my mind you should apply the default highchart settings, then run all the specific chart overrides.